app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardKey.kt

```diff
@@ -183,18 +186,53 @@ fun KeyboardKey(
                         offsetX += x
                         offsetY += y
                         if (key.slideType == SlideType.MOVE_CURSOR && slideEnabled) {
-                            if (abs(offsetX) > slideSensitivity) {
+                            if (abs(offsetY) > slideSensitivity * 5) {
+                                // If user slides upwards, enable selection
+                                if (!selection.active) {
+                                    // Activate selection
+                                    var cursorPosition =
```

Comment:
val . What IDE are you using? Android studio is giving me of a lot of easily fixible hints on your additions.